### ISSUE TITLE
vertical_parameter_alignment_on_call is triggering when it shouldn't be

### ISSUE THREAD

**Issue by tonyarnold** [realm/SwiftLint#2515]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

`vertical_parameter_alignment_on_call ` is warning when the code is already formatted as expected, ie:

```swift
        let labelRect = NSRect(
            x: titleRect.minX,
            y: titleRect.midY - (attributedLabelSize.height / 2.0) - 1.0,
            width: attributedLabel.iba_fontSizeOfFirstCharacter() * 1.5, // Fix the width — we expect only one character
            height: attributedLabelSize.height
        )
```

Produces the following warning:

```
/Users/tonyarnold/Developer/Itty Bitty Apps/reveal/Application/Source/InspectorTextFieldCell.swift:106: Vertical Parameter Alignment On Call Violation: Function parameters should be aligned vertically if they're in multiple lines in a method call. (vertical_parameter_alignment_on_call)
```
##### Complete output when running SwiftLint, including the stack trace and command used

```bash
#!/usr/bin/env bash

set -eu

SCRIPTS_DIR="${BASH_SOURCE[0]%/*}"

"${SCRIPTS_DIR}/../../Vendor/SwiftLint/swiftlint" lint --strict --config "${SCRIPTS_DIR}/../Configuration/SwiftLint.yml"
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? **0.29.1**
* Installation method used (Homebrew, CocoaPods, building from source, etc)? **Homebrew**
* Paste your configuration file:

```yml
disabled_rules:
- line_length
- nesting
- object_literal
- todo

opt_in_rules:
- array_init
- attributes
- closure_end_indentation
- closure_spacing
- conditional_returns_on_newline
- contains_over_first_not_nil
- empty_count
- empty_string
- explicit_init
- extension_access_modifier
- fatal_error_message
- file_header
- first_where
- joined_default_parameter
- let_var_whitespace
- literal_expression_end_indentation
- operator_usage_whitespace
- overridden_super_call
- pattern_matching_keywords
- prohibited_super_call
- redundant_nil_coalescing
- sorted_imports
- sorted_first_last
- switch_case_on_newline
- unneeded_parentheses_in_closure_argument
- vertical_parameter_alignment_on_call

excluded:
- Carthage
- Vendor

file_length:
    warning: 500
    error: 1000

type_name:
    max_length:
        warning: 70
        error: 100

identifier_name:
    excluded: [id, i, x, y, z, k, v]
    max_length:
        warning: 60
        error: 80

file_header:
    required_pattern: \/\/\s+Copyright © \d{4} Itty Bitty Apps Pty Ltd\. All rights reserved\.
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
  If so, paste their relative paths and respective contents.
* Which Xcode version are you using (check `xcode-select -p`)?  `/Applications/Xcode.app/Contents/Developer`
* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.

```swift
// This triggers a violation:
let foo = try! bar()
```

**Comment by tonyarnold** [comment]
I apologise for this, @jpsim - it looks like someone committed a copy of the binary to the project repo and that has been causing problems (it was version 0.27.0). Updating the binary made this issue go away. 

Total PEBKAC on my end.

**Comment by jpsim** [comment]
No worries!
